app/views/articles/show.html.erb

```diff
@@ -150,7 +150,7 @@
 
               <span class="fs-s mb-4 s:mb-0">
                 <% if @article.published_timestamp.present? %>
-                  <%= local_date(@article.published_timestamp, show_year: false) %>
+                  <%= local_date(@article.published_timestamp, show_year: @article.displayable_published_at.year != Time.current.year) %>
```

Comment:
**thought (non-blocking):** If you go with the decorator suggestion above, we could also add a `show_year?` method to that:

```ruby
local_date(@article.published_timestamp, show_year: @article.show_year?)
```

Or, potentially even better, move all of this into the decorator and this line becomes

```ruby
<%= @article.localized_timestamp %>
```

(or whatever other method name makes sense).